.github/workflows/repo-stale.yaml

```
@@ -0,0 +1,30 @@
+name: Issue Stale Check
```

```suggestion
name: Issue stale check
```

For consistency with the "Merge conflict labeler" workflow.

---

Okay, I was not sure what casing to use and too inspiration from the `App Build` Workflow :sweat_smile:
Will fix :+1: